ops/terraform/env/mgmt/iam.tf

```diff
@@ -399,3 +399,106 @@ resource "aws_iam_group_policy_attachment" "app_engineers_bfd_ssm_ro" {
   group      = aws_iam_group.app_engineers.id
   policy_arn = aws_iam_policy.bfd_ssm_ro.arn
 }
+
+# PACA / RDA
+resource "aws_iam_group" "paca_engineers" {
+  name = "bfd-paca-app-engineers"
+  path = "/"
+}
+
+resource "aws_iam_policy" "rda_ec2_instance_manager" {
+  name        = "bfd-rda-pipeline-ec2-instance-manager"
+  description = "Allow management of RDA pipeline instances"
+  path        = "/"
+  policy      = <<-POLICY
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "RDAPipelineInstanceAdmin",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:RebootInstances",
+                "ec2:TerminateInstances",
+                "ec2:StartInstances",
+                "ec2:StopInstances"
+            ],
+            "Resource": [
+                "arn:aws:ec2:us-east-1:${local.account_id}:instance/*-pipeline-rda"
+            ]
+        }
+    ]
+}
+POLICY
+}
+
+resource "aws_iam_policy" "rda_ssm_ro" {
+  description = "Allow reading RDA pipeline SSM hierarchies"
+  name        = "bfd-rda-pipeline-ssm-parameters-ro"
+  path        = "/"
+  policy      = <<-POLICY
+{
+  "Statement": [
+    {
+      "Action": [
+        "ssm:GetParametersByPath",
+        "ssm:GetParameters",
+        "ssm:GetParameter"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+        "arn:aws:ssm:us-east-1:${local.account_id}:parameter/bfd/*/pipeline/rda/*"
+      ]
```

Comment:
Note: Sid's are optional but a good practice. The version placement came along for the ride when I copied the ssm statement from earlier in the file. Some prefer the version out of the way as they very rarely ever change (there have only been two since aws inception.. one in 2008 and one in 2012).